build-system/common/check-package-manager.js

```diff
@@ -247,15 +247,20 @@ function runGulpChecks() {
     );
     updatesNeeded.add('gulp');
   } else if (!globalGulpCli) {
-    console.log(
-      yellow('WARNING: Could not find a global install of'),
-      cyan('gulp-cli') + yellow('.')
-    );
-    console.log(
-      yellow('⤷ To fix this, run'),
-      cyan('"npm install --global gulp-cli"') + yellow('.')
-    );
-    updatesNeeded.add('gulp-cli');
+    const whichGulp = getStdout('which gulp').trim();
```

Comment:
Two comments:
1. `which` is not available on Windows, so this will break our Windows developers.
2. In https://github.com/ampproject/amphtml/pull/22459, we added an explicit check for the `gulp` path because of reports that older versions in `/usr/local/bin/` were taking precedence and confusing developers with cryptic error messages. See https://github.com/ampproject/amphtml/pull/22459#issue-281441221.

That said, I think it's unacceptable for you to have to wait 10 seconds when you are using a legitimate alternative like `volta`. I'll try to make some time today to add a `volta` code path to this check so it won't bother you.

Would you mind sharing the exact path(s) where `volta` installs `gulp`, `node`, `npm`, etc. on your machine? Also, if you have a pointer to the install steps you used, that would help too.

Reply:
Here's the path the `1.0.1` version of volta uses on my machine.

`/Users/kristoferbaxter/.volta/bin/gulp`